src/components/DataDecorator/__tests__/__snapshots__/DataDecorator.spec.js.snap

```diff
@@ -165,11 +165,6 @@ exports[`DataDecorator renders correctly 1`] = `
         "WIZARD_TERTIARY_SECONDARY_TEXT": "Information will not be saved upon exit",
       }
     }
-    primaryButton={
```

Comment:
Just wanted to point out that this has been in the snapshot test for a while now, but it was the a11y & tab tests in #348 that actually caught the issue.

To me this illustrates how insufficient the snapshot tests are, due to the way they work (i.e., there is not a lot of info here & it's easy not to understand this means there's an empty button) and also how ineffectively we use them. 😅